src/pages/documentation/mobile-core/release-notes.md

```diff
@@ -30,6 +48,10 @@ Please note that the current release includes the following changes:
 * Fixed in-app message redraw on device orientation change.
 * Fixed a memory leak with the in-app message webview.
 
+### Android Lifecycle 2.0.2
+
+* Chain Lifecycle events.
```

Comment:
can we make this a little more descriptive?  what does it mean?  what is the purpose?